File: java/PerangkatJaringan.java
public class PerangkatJaringan extends Komponen {
    private String alamatIp;
    private String nameProvider;
    private String internet;

    public PerangkatJaringan(String merk, String nama, String alamatIp, String nameProvider, String internet) {
        super(merk, nama);
        this.alamatIp = alamatIp;
        this.nameProvider = nameProvider;
        this.internet = internet;
    }

    // Getter
    public String getAlamatIp() {
        return alamatIp;
    }

    public String getNameProvider() {
        return nameProvider;
    }

    public String getInternet() {
        return internet;
    }

    // Setter
    public void setAlamatIp(String alamatIp) {
        this.alamatIp = alamatIp;
    }

    public void setNameProvider(String nameProvider) {
        this.nameProvider = nameProvider;
    }

    public void setInternet(String internet) {
        this.internet = internet;
    }

    // Method info
    public String info() {
        return "Perangkat Jaringan: " + getNama() + " (Merk: " + getMerk() + ")\n" +
               "  IP: " + alamatIp + "\n" +
               "  Provider: " + nameProvider + "\n" +
               "  Internet: " + internet;
    }
}

File: Python/PerangkatJaringan.py
from Komponen import Komponen


class PerangkatJaringan(Komponen):
    def __init__(self, merk="", nama="", alamat_ip="", name_provider="", internet=""):
        super().__init__(merk, nama)
        self.alamat_ip = alamat_ip
        self.name_provider = name_provider
        self.internet = internet

    # Getter
    def get_alamat_ip(self):
        return self.alamat_ip

    def get_name_provider(self):
        return self.name_provider

    def get_internet(self):
        return self.internet

    # Setter
    def set_alamat_ip(self, alamat_ip):
        self.alamat_ip = alamat_ip

    def set_name_provider(self, name_provider):
        self.name_provider = name_provider

    def set_internet(self, internet):
        self.internet = internet

    # Method info
    def info(self):
        return (f"Perangkat Jaringan: {self.get_nama()} (Merk: {self.get_merk()})\n"
                f"  IP: {self.alamat_ip}\n"
                f"  Provider: {self.name_provider}\n"
                f"  Internet: {self.internet}")